pkg/rid/cockroach/store.go

```diff
@@ -182,6 +182,24 @@ func (s *Store) GetVersion(ctx context.Context) (string, error) {
 		return "v1.0.0", nil
 	}
 	// Version without cells joins table.
+
+	// We treat the existence of scd_constraints as running on the
+	// newest version, 3.0.0
+	const query3 = `
```

Comment:
This is really polluting the api.. i know you're hoping to wait for the schema management PR, but we could accomplish this with 2 separate versions, and 2 separate calls to the DB if we decide not to

Reply:
This PR has been updated to use the recently-developed migrations (in particular, applying them to SCD in #353).